app/components/UI/FiatOnRampAggregator/components/TransactionDetails.tsx

```diff
@@ -260,7 +293,7 @@ const TransactionDetails: React.FC<Props> = ({ order }: Props) => {
 				<Text small>{strings('fiat_on_ramp_aggregator.transaction.questions')} </Text>
 				<TouchableOpacity>
 					<Text small underline>
-						{strings('fiat_on_ramp_aggregator.transaction.contact')} {order.providerName}{' '}
+						{strings('fiat_on_ramp_aggregator.transaction.contact')} {order.data.provider.name}{' '}
```

Comment:
use `getProviderName` so we can have a single source of truth for these names
https://github.com/MetaMask/metamask-mobile/blob/e632be20eac704ee6bbfd9a0713d0e3f49e54c43/app/reducers/fiatOrders/index.js#L51